awx/api/serializers.py

```diff
@@ -2734,7 +2734,7 @@ class Meta:
         fields = ('*', 'job_type', 'inventory', 'project', 'playbook', 'scm_branch',
                   'forks', 'limit', 'verbosity', 'extra_vars', 'job_tags',
                   'force_handlers', 'skip_tags', 'start_at_task', 'timeout',
-                  'use_fact_cache', 'organization',)
+                  'use_fact_cache', 'organization', 'limit',)
```

Comment:
this is duplicated with its entry in the tuple 2 lines above here.

Reply:
I actually just tested this without my duplication and it's already working the way we had hoped in the issue. I'm gonna close this and the bug, since it seems it's already resolved.